### PR TITLE
Add example for `SHOW TRANSACTIONS` with durations

### PR DESCRIPTION
**Pull request by Hunterness**

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TransactionsCommandTest.scala [hunk 1]
```diff
@@ -216,6 +216,26 @@ All users may view all of their own currently executing transactions.
             resultTable()
           }
         }
+        p(
+          """Several of the output columns have the `duration` type which can be hard to read.
+            #They can instead be returned in more readable format:""".stripMargin('#'))
```
Comment (phil198) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TransactionsCommandTest.scala]:
```suggestion
            #They can instead be returned in a more readable format:""".stripMargin('#'))
```

**Review by phil198 — APPROVED** [review]

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TransactionsCommandTest.scala [hunk 2]
```diff
@@ -216,6 +216,26 @@ All users may view all of their own currently executing transactions.
             resultTable()
           }
         }
+        p(
+          """Several of the output columns have the `duration` type which can be hard to read.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TransactionsCommandTest.scala]:
```suggestion
          """Several of the output columns have the `duration` type, which can be hard to read.
```

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TransactionsCommandTest.scala (hunk 1 above)
Comment (AlexicaWright) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TransactionsCommandTest.scala]: +1

**Review by AlexicaWright — COMMENTED** [review]
Just a little punctuation :)

**Review by AlexicaWright — APPROVED** [review]
👍

**Pull request merged**